### ISSUE TITLE
Reflect.get called on non-object when initializing in-memory store

### ISSUE THREAD

**Issue by tpluscode** [oxigraph/oxigraph#942]
When using the WASM build, I found that I could not create a store using this code

```js
import oxigraph from 'oxigraph'
import rdf from '@zazuko/env-node'

(async function() {
  const dataset = await rdf.dataset().import(rdf.fromFile('cas.ttl'))
  const store = new oxigraph.Store([...dataset])
})()
```

On the other hand, adding the quads one-by-one works

```js
import oxigraph from 'oxigraph'
import rdf from '@zazuko/env-node'

(async function() {
  const dataset = await rdf.dataset().import(rdf.fromFile('cas.ttl'))
  const store = new oxigraph.Store()
  for (const quad of dataset) {
    store.add(quad)
  }
})()
```

Here's the file in question: [cas.ttl.zip](https://github.com/user-attachments/files/16402974/cas.ttl.zip). Fairly big but I was unable to determine if any particular quad was responsible.

**Comment by Tpt** [comment]
This error seems to be caused by a bug in [wasm-bindgen reference-type support](https://rustwasm.github.io/docs/wasm-bindgen/reference/reference-types.html). I have circumvented it by using iteration inside of `Store` constructor instead of conversion between a JS array and Rust array: #947

**Comment by Tpt** [comment]
Released as part of 0.4.0-alpha.8
